Source:   http://websvn.kde.org/branches/KDE/4.4/kdebindings/python/pykde4/sip/akonadi/addressattribute.sip?r1=1070858&r2=1150377&view=patch
Upstream: yes
Reason:   Akonadi bindings don't build with sip-4.10.3 and higher

--- kdebindings/python/pykde4/sip/akonadi/addressattribute.sip	2010/01/06 22:46:33	1070858
+++ kdebindings/python/pykde4/sip/akonadi/addressattribute.sip	2010/07/15 20:19:01	1150377
@@ -57,6 +57,10 @@
     else if (dynamic_cast<Akonadi::MessageThreadingAttribute*>(sipCpp))
         sipType = sipType_Akonadi_MessageThreadingAttribute;
 %End
+private:
+//force
+Akonadi::AddressAttribute& operator = (const Akonadi::AddressAttribute&);
+//end
 };
 };
 %ModuleHeaderCode
